### DISCUSSION TITLE
No examples or documentation about unit testing in vuetify 3 beta

### DISCUSSION THREAD

**Discussion by sulhome** [vuetifyjs/vuetify#15413] [Q&A]
I have been struggling to run simple unit test example from the auto generated Vuetify 3.
Can you please include documentation on how to unit test vue 3 components which has vuetify 3 components.

You can run a test with a warning but in order to interact with vuetify components you need them to render, basic stuff like button for example.
I have also noticed lack of documentation on typescript. For example if I setup vue to use typescript how can I run a simple unit test which includes vuetify component?

Running a simple test with text field give this warning:
``` 
console.warn
[Vue warn]: Failed to resolve component: v-text-field
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
      at <HomeView ref="VTU_COMPONENT" > 
      at <VTUROOT>
 ```
 If this warning go away then vuetify components will render. Again couldn't find anything to advise how to setup this properly not in JavaScript or TypeScript. In fact there is a unit testing option in vuetify 2 version documentation under getting started but none of that on vuetify 3 unless I am searching in the wrong menu

Thank you

**Comment by rangeldor** [comment]
I did a simple test too like you, and I got this error, then I used this:

```ts 
import { createVuetify } from 'vuetify'
import * as components from 'vuetify/components'
import * as directives from 'vuetify/directives'

describe('TodoList.vue', () => {
  const vuetify = createVuetify({ components, directives })
...

const wrapper = shallowMount(TodoList, {
      global: {
        plugins: [pinia, vuetify]
      }
})

```

But I got this error:

 import { createDefaults, DefaultsSymbol } from "./composables/defaults.mjs";
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
